code/modules/mod/modules/modules_ninja.dm

```diff
@@ -174,7 +174,13 @@
 	/// The item linked to the module that will get recalled.
 	var/obj/item/linked_weapon
 	/// The accepted typepath we can link to.
-	var/accepted_type = /obj/item/energy_katana
+	var/list/accepted_types = list(
```

Comment:
this module's description is written around the energy katana, you need to rewrite it to make sense